### ISSUE TITLE
apinger: breaking line syntax

### ISSUE THREAD

**Issue by 1977er** [freifunkh/ansible#271]
Seen in logs:
`telegraf[1264]: 2024-10-05T09:49:30Z E! [inputs.exec] Error in plugin: metric parse error: expected field at 2:60: "apinger,description=sn07,target=10.10.70.1,proto=ipv4 loss= 1728121770000000000"`

Missing `loss` field.

Either fix awk script in `/usr/bin/ffh_apinger_stats.sh` or mangle the apinger output in `/tmp/apinger.status` otherwise.

**Comment by 1977er** [comment]
Quite a non issue. Happens only once in a few weeks. Won't fix until it gets a real issue.
